modules/node_modules/@ncigdc/components/DownloadButton.js

```diff
@@ -28,14 +31,20 @@ const DownloadButton = ({
   filters = {},
   active,
   setActive,
+  altMessage = false,
+  style = {},
   extraParams = {},
+  dispatch,
 }) => {
-  const icon = <i key="icon" className={`fa fa-download ${active ? 'fa-spinner fa-pulse' : ''}`} />;
+  const icon = active ? <Spinner /> : <DownloadIcon />;
```

Comment:
`aria-label` should also work, although that would go on the button and not the icon
As a tech director of ours was fond of saying, there's more than one way to skin a cat :)